app/src/main/java/io/left/meshim/models/Message.java

```diff
@@ -137,10 +146,12 @@ public Message(User sender, User recipient, String message) {
      * @param isMyMessage if this device's user sent the message
      */
     @Ignore
-    public Message(User sender, User recipient, String message, boolean isMyMessage) {
+    public Message(User sender, User recipient, String message, boolean isMyMessage, String filePath, String fileName) {
         this(sender, sender.id, recipient, recipient.id, message, isMyMessage, new Date());
         this.isRead = false;
         this.isDelivered =false;
+        this.fileName = fileName;
```

Comment:
If all constructors eventually bubble up to the constructor on line 167, we don't need to set these defaults here, they'll get set at the top level.